Source/GSCSVariable.m

```diff
@@ -24,4 +24,146 @@
 
 @implementation GSCSVariable
 
+- (instancetype) init
+{
+  self = [super init];
```

Comment:
In standard ObjC these additional init methods would call the dedicated initialiser `-initWithName:type:`

Reply:
Fixed